include/net-snmp/types.h

```diff
@@ -299,7 +299,7 @@ struct snmp_session {
     long            version;
     /** Number of retries before timeout. */
     int             retries;
-    /** Number of uS until first timeout, then exponential backoff */
+    /** Message timeout in uS before first retry and between retries */
```

Comment:
uS = microSiemens (electrical conductance). Please change `uS` into `us` or `μs`. See also https://en.wikipedia.org/wiki/International_System_of_Units.

Reply:
Updated, although I personally prefer to set my message timeouts in microSiemens :wink: